### PR TITLE
Create Home page

### PR DESCRIPTION
**Pull request by IgorOsiyuk**
create redux store
action and reducer for login 
for Authorized user show the main page as a Keep 
image

for Unauthorized user show the main page  with buttons "Registration" "Login" 
image

Link https://i-tech-art-laba-front-jcp27oivv-igorosiyuk.vercel.app/

### CODE REVIEW

File: google_keeps/src/redux/action/types.jsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export const LOGIN_USER = "LOGIN_USER";
```
Comment (melonholy) [inline google_keeps/src/redux/action/types.jsx:1]: new line

File: google_keeps/src/components/pages/Main.jsx [hunk 2]
```diff
@@ -14,7 +18,16 @@ const MainPageWrapper = styled.div`
 `;
 
 const Main = () => {
-  return (
+  const dispatch = useDispatch();
+  const isLogin = useSelector((state) => state.auth.isLogin);
```
Comment (melonholy) [inline google_keeps/src/components/pages/Main.jsx:22]: params in useSelector moving in separate folder named `selectors`

File: google_keeps/src/components/pages/Main.jsx [hunk 3]
```diff
@@ -14,7 +18,16 @@ const MainPageWrapper = styled.div`
...
+  const user = AuthService.getcurrentUser();
+  if (user) {
+    dispatch(LoginAction());
+  }
+
```
Comment (melonholy) [inline google_keeps/src/components/pages/Main.jsx:27]: Wrap it in useEffect

**Review by melonholy — CHANGES_REQUESTED** [review]

File: google_keeps/src/components/pages/Main.jsx (hunk 2 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/pages/Main.jsx]: - [+]

File: google_keeps/src/components/pages/Main.jsx (hunk 3 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/pages/Main.jsx]: - [+]

File: google_keeps/src/redux/action/types.jsx (hunk 1 above)
Comment (IgorOsiyuk) [reply google_keeps/src/redux/action/types.jsx]: - [+]

Comment (IgorOsiyuk) [reply google_keeps/src/redux/action/types.jsx]: - [+]

File: google_keeps/src/components/pages/Main.jsx (hunk 2 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/pages/Main.jsx]: - [+]

File: google_keeps/src/components/pages/Main.jsx (hunk 3 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/pages/Main.jsx]: - [+]

**Review by melonholy — APPROVED** [review]

**Pull request merged**